internal/playlist/details.go

```diff
@@ -0,0 +1,56 @@
+package playlist
+
+import (
+	"errors"
+	"fmt"
+	"spotify/internal"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func NewDetailsCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:     "details",
+		Aliases: []string{"e"},
+		Short:   "Get information on a playlist",
+		Args:    cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			api, err := internal.Authenticate()
+			if err != nil {
+				return err
+			}
```

Comment:
Basically everything under this comment needs to be put in a function called something like `Show()`, which takes the `api` as an argument so it can be tested.